src/pages/en/guides/cms/storyblok.md

```diff
@@ -2,13 +2,215 @@
+
+    :::note
+    The component paths are relative to the `src` directory.
+    :::
+
+3. `apiOptions` - An object containing [Storyblok API options](https://github.com/storyblok/storyblok-astro#options). 
+
+    :::caution
+    The default region is `eu`. If your Storyblok space was created in the US region, you will need to set the `region: 'us'`.
+    :::
+
+### Connecting bloks to Astro components
+
+To connect your bloks to Astro, create a new folder named `storyblok` in the `src` directory. This folder will contain all the Astro components that will match your bloks in your Storyblok blok library.
+
+In this example, we have a `blogPost` blok content type in our Storyblok library with the following fields:
+
+- `title` - A text field
+- `description` - A text field
+- `content` - A rich text field
+
+Our goal is to create the equivalent Astro component that will use these fields to render its content. To do this, create a new file named `BlogPost.astro` inside `src/storyblok` with the following content:
+
+```astro title="src/storyblok/BlogPost.astro"
+---
+import { storyblokEditable, renderRichText } from '@storyblok/astro'
+
+const { blok } = Astro.props
+const content = renderRichText(blok.content)
+---
+
+<article {...storyblokEditable(blok)}>
+  <h1>{blok.title}</h1>
+  <p>{blok.description}</p>
+  <Fragment set:html={content} />
+</article>
+```
+
+The `blok` property is the data that we will receive from Storyblok. It will contain the values of the fields that we defined in the previous step.
+
+To render our content, the integration provides utility functions such as:
+
+- `storyblokEditable` - it adds the necessary attributes to the elements so that you can edit them in Storyblok.
+- `renderRichText` - it transforms the rich text field into HTML.
+
+Your root directory should include these new files:
+
+```ini title="Project Structure" ins={3}
+├── src/
+│   └── storyblok/
+│       └── BlogPost.astro
+├── .env
+├── astro.config.mjs
+└── package.json
+```
+
+Finally, to connect the `blogPost` blok to the `BlogPost` component, add a new property to your components object in your Astro config file. The key is the name of the blok and the value is the path to the component.
+
+```js title="astro.config.mjs" ins={15}
+import { defineConfig } from 'astro/config';
+import storyblok from '@storyblok/astro';
+import { loadEnv } from 'vite';
+
+const env = loadEnv(import.meta.env.MODE, process.cwd(), 'STORYBLOK');
+
+export default defineConfig({
+  integrations: [
+    storyblok({
+      accessToken:
+        import.meta.env.MODE === 'development'
+          ? env.STORYBLOK_PREVIEW_TOKEN
+          : env.STORYBLOK_PUBLIC_TOKEN,
+      components: {
+        blogPost: 'storyblok/BlogPost',
+      },
+      apiOptions: { 
+        region: 'us',
+      },
+    })
+  ],
+});
+```
+To test the setup, create a new story with the `blogPost` content type. Then fetch and render your story directly into a page:
```

Comment:
maybe we can label this small section as fetching data? What do you think @Jutanium ?
```suggestion
### Fetching data

To test the setup, create a new story with the `blogPost` content type. Then fetch and render your story directly into a page:
```

Reply:
I agree, since it's no longer about connecting components!